Library/Homebrew/cask_dependent.rb

```diff
@@ -16,7 +16,9 @@ def full_name
   end
 
   def runtime_dependencies(ignore_missing: false)
-    recursive_dependencies ignore_missing: ignore_missing
+    recursive_dependencies(ignore_missing: ignore_missing).select do |dependency|
+      dependency.tags.blank?
```

Comment:
Do we support `:optional` or `:recommended` here?

Reply:
Or `:test`?

Reply:
I think `runtime_dependents` shouldn't include `:test` dependents, but you are right that my suggestion didn't take into account `:optional` and `:recommended`. Perhaps we can modify this to reject `:build` and `:test` instead.

Reply:
Sounds good!